src/EventSubscriber/GeneratedUserIngestControllerSubscriber.php

```diff
@@ -58,8 +59,8 @@ public function __construct(
         LineItemRepository $lineItemRepository,
         FolderSyncService $userFolderSync,
         CreateUserServiceContext $createUserServiceContext,
-        bool $enabled,
-        string $group
+        UserGeneratorParametersBag $parametersBag,
```

Comment:
`UserGeneratorParametersBag` does exactly the same as already existing `CreateUserServiceContext`
please remove one of those